src/test/java/ca/gc/cyber/ops/assemblyline/java/client/clients/AssemblylineClientTest.java

```diff
@@ -585,8 +538,19 @@ void testIngestGetMessageList() {
         mockResponse(MockResponseModels.getIngestMessageListJson());
 
         verifyHttpGet(this.assemblylineClient.getIngestMessageList("test_java_client"),
-                "/api/v4/ingest/get_message_list/test_java_client/",
-                MockResponseModels.getIngestMessageList());
+                "/api/v4/ingest/get_message_list/test_java_client/", MockResponseModels.getIngestMessageList());
+    }
+
+    @Test
+    void testConstructor_validProxyHostAndPort() {
```

Comment:
We have not used underscores in any other of the method names so I would stick to strick camelCase to follow convention

Reply:
changed